drivers/wifi/nrf700x/zephyr/inc/zephyr_fmac_coex.h

```diff
@@ -0,0 +1,73 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/**
+ * @brief Header containing Coexistence APIs.
+ */
+
+#ifndef __ZEPHYR_FMAC_COEX_H__
+#define __ZEPHYR_FMAC_COEX_H__
+
+/* Indicates WLAN frequency band of operation */
+enum {
+	WLAN_5G = 0,
+	WLAN_2PT4G
+};
+
+/**
+ * @function   nrf_wifi_coex_config_pta(void)
+ *
+ * @brief      Function used to configure PTA tables of coexistence hardware.
+ *
+ * @param[in]  wlan_band
+ *             Indicates WLAN's frequency band of operation.
+ *             Set CONFIG_COEX_WLAN_2PT4G=y in prj.conf if 2.4GHz.
+ *             Set CONFIG_COEX_WLAN_2PT4G=n if 5GHz.
+ * @return     Returns status of configuration.
+ *             Returns zero upon successful configuration.
+ *             Returns non-zero upon unsuccessful configuration.
+ */
+int nrf_wifi_coex_config_pta(uint32_t wlan_band);
+
+
+/**
+ * @function   nrf_wifi_config_sr_switch()
+ *
+ * @brief      Function used to configure SR side switch (nRF5340 side switch in nRF7002 DK).
```

Comment:
Explain that this API configures the SR side antenna switch as per the selection of shared/separate antenna mentioning CONFIG_COEX_SEP_ANTENNAS symbol needs to be set appropriately.